docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md

```diff
@@ -0,0 +1,95 @@
+# BigDL-Nano PyTorch Overview
+
+BigDL-Nano can be used to accelerate PyTorch or PyTorch-Lightning applications on both training and inference workloads. The optimizations in BigDL-Nano are delivered through an extended version of PyTorch-Lightning `Trainer`. These optimizations are either enabled by default or can be easily turned on by setting a parameter or calling a method.
+
+## PyTorch Training
+
+We will briefly describe here the major features in BigDL-Nano for PyTorch training. You can find complete examples here [links to be added]().
+
+### Best Known Configurations
+
+When you run `source bigdl-nano-init`, BigDL-Nano will export a few environment variables, such as OMP_NUM_THREADS and KMP_AFFINITY, according to your current hardware. Empirically, these environment variables work best for most PyTorch applications. After setting these environment variables, you can just run your applications as usual (`python app.py`) and no additional changes are required.
+
+### BigDL-Nano PyTorch Trainer
+
+The PyTorch Trainer (`bigdl.nano.pytorch.Trainer`) is the place where we integrate most optimizations. It extends PyTorch Lightning's Trainer and has a few more parameters and methods specific to BigDL-Nano. The Trainer can be directly used to train a `LightningModule`.
```

Comment:
We should import `bigdl.nano.pytorch.Trainer` here: https://github.com/intel-analytics/BigDL/blob/main/python/nano/src/bigdl/nano/pytorch/__init__.py

Reply:
raised an issue here: #4604

Reply:
added the "import" in `bigdl.nano.pytorch`